### ISSUE TITLE
The data rates in the ChromiumNetworkConditions.cs properties are incorrect

### ISSUE THREAD

**Issue by TridentTrue** [SeleniumHQ/selenium#9461]
## 🐛 Bug Report
The data rate in the tooltips for [DownloadThroughput](https://github.com/SeleniumHQ/selenium/blob/0436178e2567ae5f5e3b6ff2c7a5006a1001bd2b/dotnet/src/webdriver/Chromium/ChromiumNetworkConditions.cs#L55) and [UploadThroughput](https://github.com/SeleniumHQ/selenium/blob/0436178e2567ae5f5e3b6ff2c7a5006a1001bd2b/dotnet/src/webdriver/Chromium/ChromiumNetworkConditions.cs#L64) of ChromiumNetworkConditions.cs appear to be incorrectly listed as kb/second when it should be bytes per second.

## To Reproduce
For any basic test using the `ChromeDriver`, initialize the driver's `NetworkConditions` property with a `ChromeNetworkConditions` object with `DownloadThroughput` and `UploadThroughput` in kb/s, then run the test while observing data transfer rate using your preffered monitoring software e.g. WireShark

## Expected behavior
Chrome driver throttles network rates at the speeds that were defined in kb/s.

## Test script or set of commands reproducing this issue
If for example you supply the download with a value of 10000 then during the actual test it will show a rate of 0.08Mbps rather than the expected 10Mbps.

To get the desired 10Mbps throughput you would have to instead set the download value to 1250000.
```cs
public class NetworkConditionsTests
{
    public void IncorrectNetworkThrottleTest()
    {
        int latencyInMilliseconds = 20;
        long downloadLimitMbps = 10;
        long uploadLimitMbps = 5;

        ChromeDriver driver = new ChromeDriver()
        {
            NetworkConditions = new ChromeNetworkConditions()
            {
                Latency = new TimeSpan(0, 0, 0, 0, latencyInMilliseconds),
                DownloadThroughput = downloadLimitMbps * 1000, // Mbps to kb/s
                UploadThroughput = uploadLimitMbps * 1000, // Mbps to kb/s
                IsOffline = false,
            }
        };

        driver.Navigate().GoToUrl("https://github.com/SeleniumHQ/selenium");

        var wait = new WebDriverWait(driver, new TimeSpan(0, 0, 30));
        wait.Until(SeleniumExtras.WaitHelpers.ExpectedConditions.ElementIsVisible(By.ClassName("repository-content")));
    }

    public void CorrectNetworkThrottleTest()
    {
        int latencyInMilliseconds = 20;
        long downloadLimitMbps = 10;
        long uploadLimitMbps = 5;

        ChromeDriver driver = new ChromeDriver()
        {
            NetworkConditions = new ChromeNetworkConditions()
            {
                Latency = new TimeSpan(0, 0, 0, 0, latencyInMilliseconds),
                DownloadThroughput = downloadLimitMbps * 125000, // Mbps to bytes per second
                UploadThroughput = uploadLimitMbps * 125000, // Mbps to bytes per second
                IsOffline = false,
            }
        };

        driver.Navigate().GoToUrl("https://github.com/SeleniumHQ/selenium");

        var wait = new WebDriverWait(driver, new TimeSpan(0, 0, 30));
        wait.Until(SeleniumExtras.WaitHelpers.ExpectedConditions.ElementIsVisible(By.ClassName("repository-content")));
    }
}
```

## Environment

OS: Windows 10
Browser: Google Chrome
Browser version: 89.0.4389.2300
Browser Driver version: ChromeDriver 89.0.4389.2300
Language Bindings version: C# 3.141.0
Selenium Grid version (if applicable): N/A

**Comment by jimevans** [comment]
@TridentTrue What's the actual issue here, that the IntelliSense documentation comments say the values are defined as kb/s; or that the command should be using kb/s as documented and implemented in the chromedriver source are using bytes/s?

**Comment by TridentTrue** [comment]
@jimevans I would imagine it's more likely that the intellisense is incorrect rather than the actual implementation, especially if it's just plugging the parameters straight into the chrome API. May also break some users existing tests if the numbers were tweaked but I don't know how heavily the feature is used.

**Comment by jimevans** [comment]
Fixed doc string in cd12f66.
